dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:16.04
+
```

Comment:
I have no idea why do we need Dockerfile at this repo. Do you want perform some tests? Feel free to create a _local_ entrypoint script and pass to container instance using `--entrypoint`